csharp-refactoring/RefactoringObjectOrientationAbusers/RefactoringObjectOrientationAbusers/RefusedBequest/UseInheritance/Correct/Animal.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace RefactoringObjectOrientationAbusers.RefusedBequest.UseInheritance.Correct
+{
+    public class Animal
+    {
+        public void Eat()
+        {
+            Console.WriteLine("Eating...");
+        }
+
+        public void Sleep()
+        {
+            Console.WriteLine("Sleeping...");
+        }
+    }
+}
```

Comment:
Altough this looks correct, in the article the screenshot looks slightly different. Can you update the screenshot in the article?